### DISCUSSION TITLE
How to include an icon in a dashboard nav item?

### DISCUSSION THREAD

**Discussion by pensivedog** [quarto-dev/quarto-cli#11605] [Q&A] [answered]
### Description

Is it possible to get a bootstrap icon in a dashboard navigation tab? Pasting an emoji works, but I've tried everything I can think of and can't get a bs icon to appear in the dashboard nav alongside an item's text.

```
---
title: "Dashboard with Icons in Nav"
format: dashboard
---

# Nav tab 1 {how to add a bootstrap icon here?}

## Row

# "Or here?" Nav tab 2

```

**Comment by mcanouil** [comment]
either you know the HTML code or you use shortcode from an extension

```
quarto add mcanouil/quarto-iconify
```

````qmd
---
title: "Dashboard with Icons in Nav"
format: dashboard
---

# Nav tab 1 {{< iconify iconSet:iconName >}}

## Row

# `<CODE>`{=html} Nav tab 2
````

**Reply by pensivedog** [reply]
Thank you for the reply. I'm trying both of these methods and somehow it's not working. Could it be related to my dashboard living in a subdirectory of a quarto website? This means that above the dashboard nav there is the website nav. Could that be interfering with html rendering in the dashboard nav items?

The icons appear fine in normal markdown content, just not in the nav item content.

**Reply by mcanouil** [reply]
Indeed, all html/formatting is removed of the titles.

```
---
title: "Dashboard with Icons in Nav"
format: dashboard
---

# Nav **tab 1** [text]{style="color: red;"}
```

I can't tell if this is intentional or not.

**Reply by pensivedog** [reply]
Alright. Thanks for confirming it's not just happening on my end.

It's pretty normal to want to put icons in nav menus... I hope this can somehow get addressed. Or, if anyone has a workaround in the meantime, please share!

**Reply by mcanouil** [reply]
At this point, it's not a trivial workaround.

@cderv @cscheid do you know if it is by design that dashboard markdown headers are striped out of any formatting/markdown?

**Reply by cscheid** [reply]
Unlikely to be by design, but a simplifying assumption. Good `enhancement` ticket material.

**Reply by mcanouil** [reply, marked as answer]
- https://github.com/quarto-dev/quarto-cli/issues/11634
